calicoctl/commands/node/run.go

```diff
@@ -141,6 +142,10 @@ Options:
                            within the Docker containers that have been networked
                            by the Calico driver.
                            [default: cali]
+     --enable-docker-networking-labeling
+                           Enables adding labels to Docker containers that can
+                           be used in policies. Setting this flag also disables
+                           the default network profiles.
```

Comment:
Not sure what a "default network profile" is - I assume that this also disables adding a network-specific Calico profile